app/views/play/level/modal/CourseVictoryModal.coffee

```diff
@@ -106,6 +106,8 @@ module.exports = class CourseVictoryModal extends ModalView
       # TODO: use supermodel.loadCollection for better caching but watch out for @session overwriting
       @levelSessions = new LevelSessions()
       @levelSessions.fetchForCourseInstance(@courseInstanceID, {}).then(=> @levelSessionsLoaded())
+    else
+      @levelSessionsLoaded()
```

Comment:
Teacher in course view has no session, thus the `if` statement fails and `@levelSessionsLoaded` is never called.
This else statement covers teachers playing using the course guide.